EnglishFilter/sections/adservers.txt

```diff
@@ -152,6 +152,7 @@
 ||putfeepea.site^$third-party
 ||gagpetwet.com^$third-party
 ||wastesshimssat.world^$third-party
+||hqr9q6f30371.com^
```

Comment:
why without `^$third-party`?

Reply:
because site doesn't exist on this domain